src/Microsoft.Identity.Client/Event.cs

```diff
@@ -58,5 +61,31 @@ public void Stop()
         {
             this[ElapsedTime] = (CurrentUnixTimeMilliseconds() - _startTimestamp).ToString();  // It is a duration
         }
+
+        public static string ScrubTenant(Uri uri) // Note: There is also a Unit Test case for this helper
```

Comment:
It was there, until later I realized that the upcoming ApiEvent will also need to use this helper, so the natural reaction is to promote it into the base class, so that it can be accessed by 2 sub-classes.

Bottom line, it is functionally equivalent either way.